diff --git a/dist/sandbox/macos-sandbox-utils.js b/dist/sandbox/macos-sandbox-utils.js
index 0e9e58a..d3b2137 100644
--- a/dist/sandbox/macos-sandbox-utils.js
+++ b/dist/sandbox/macos-sandbox-utils.js
@@ -279,6 +279,7 @@ function generateSandboxProfile({ readConfig, writeConfig, httpProxyPort, socksP
         '  (global-name "com.apple.audio.systemsoundserver")',
         '  (global-name "com.apple.distributed_notifications@Uv3")',
         '  (global-name "com.apple.FontObjectsServer")',
+        '  (global-name "com.apple.FSEvents")',
         '  (global-name "com.apple.fonts")',
         '  (global-name "com.apple.logd")',
         '  (global-name "com.apple.lsd.mapdb")',
